nixos/modules/programs/nano.nix

```diff
@@ -12,16 +12,16 @@ in
 
       nanorc = lib.mkOption {
         type = lib.types.lines;
-        default = "";
+        default = ''
+          set nowrap
+          set tabstospaces
+          set tabsize 2
+          include "${pkgs.nano}/share/nano/*.nanorc"
```

Comment:
@joachifm 
> Perhaps enabling syntax highlighting could be made into an option?

Added, thank you!

> Note that now nix must build nano to generate the manual; usually, we use defaultText to work around that.

Should I just put `defaultText = "something";`? I see manual gets rebuilded only when I change default value in module file, not when I overwrite `programs.nano.nanorc` in configuration.nix